.clang-format

```
@@ -17,4 +17,9 @@ AllowShortLoopsOnASingleLine: false
 BinPackParameters: false
 BreakBeforeBinaryOperators: true
 BreakBeforeTernaryOperators: true
-ContinuationIndentWidth: 4
+ContinuationIndentWidth: 2
+AlignEscapedNewlines: DontAlign
+Cpp11BracedListStyle: false
+IncludeCategories:
```

Might as well be explicit where we can, at least in this case where its presence near  IncludeCategories is a hint.

```suggestion
SortIncludes: true
IncludeCategories:
```